app/index.js

```diff
@@ -82,6 +82,7 @@ const basicAuth = require('./browser/basicAuth')
 const async = require('async')
 const tabs = require('./browser/tabs')
 const settings = require('../js/constants/settings')
+require('./browser/webtorrent')
```

Comment:
`patchUserDataDir` does it the same way, but sure, i can give our module an init function